README.md

```diff
@@ -396,10 +396,20 @@ sudo ./check-production.sh
 	Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>
```

Comment:
Please change the description further above to better integrate your additions.
Change "Please check the service does not output any error message:" to "Please check the following two logs to confirm successful registration:".
Then, add a number list and the first point is: "Check service log with following command:"